ssdaq/core/ss_data_classes.py

```diff
@@ -8,6 +8,19 @@
 N_BYTES_NUM = 8 #Number of bytes to encode numbers (uint and float) in the SSReadout
 N_CAM_PIX = N_TM*N_TM_PIX #Number of pixels in the camera
 
+from collections import namedtuple as _nt
+ 
+_SSMappings = _nt('SSMappings','ssl2colrow ssl2asic_ch')
```

Comment:
What is the reason for creating the `ssl2colrow` mapping? If the data ordering changes in CHEC, then that means two mapping arrays need to be updated in this package instead of one.

Would it not be better to only have the `ssl2asic_ch` mapping, and then rely on the common mapping classes from TargetCalib?

Reply:
The reason I kept the col-row mapping is because it provides a simple mapping of the SS lines to cols and rows as they are ordered physically on the tile. This makes it easy to plot the data of a single TM without having to go through yet another mapping as is with the case of ASIC-CHANNEL

Reply:
Okay, I guess when we have a centralised mapping library this code can be simplified